src/Modules/Profile/Commands.Profile/ConnectPowerBIServiceAccount.cs

```diff
@@ -13,7 +13,7 @@
 namespace Microsoft.PowerBI.Commands.Profile
 {
     [Cmdlet(CmdletVerb, CmdletName, DefaultParameterSetName = UserParameterSet)]
-    [Alias("Login-PowerBIServiceAccount", "Set-PowerBIServiceAccount", "Add-PowerBIServiceAccount")]
+    [Alias("Login-PowerBIServiceAccount", "Set-PowerBIServiceAccount", "Add-PowerBIServiceAccount", "Login-PowerBI")]
```

Comment:
I think Kay also mentioned renaming the non-alias commands.
Looking at the PowerShell verbs list, what do you think of Connect-PowerBI and Disconnect-PowerBI?

Also, just to nitpick, I find the Add/Remove aliases odd here.

Reply:
We can do that in a separate PR\release as it's a larger change. We can ensure compat by creating aliases for the older names